ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_sorting.py

```diff
@@ -108,3 +106,38 @@ def test_numpy_msort(
         on_device=on_device,
         a=x[0],
     )
+    
+
+@handle_frontend_test(
+    fn_tree="numpy.sort_complex",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+        min_dim_size=1,
+        min_axis=-1,
+        max_axis=0,
+    ),
+)
+def test_numpy_sort_complex(
+        *,
+        dtype_x_axis,
+        as_variable,
+        with_out=False,
```

Comment:
just remove this entirely & hardcode the `with_out` in `test_frontend_function`

Reply:
If I remove "dtype_x_axis" and "as_variable" it would effect the lines in which I have used them.